### ISSUE TITLE
"Not in the main menu. Press ESC in-game and rerun the scanner."

### ISSUE THREAD

**Issue by GuyInDogSuit** [Psycho-Marcus/WuWa_Inventory_Kamera#2]
No matter what I do, I get this error every time. My resolution is 1920x1080, and I even unplugged my controller. Didn't start the app until I was in-game in the terminal screen. Still no dice. I don't understand why. I just want to scan my achievements.

**Comment by Psycho-Marcus** [comment]
Are both your system resolution and in-game resolution set to 1920x1080? Also, please try using the new release.

**Comment by GuyInDogSuit** [comment]
Yes, both are 1920x1080. I have a dual-monitor setup but even with just one on, the issue persists.
